requirements.txt

```diff
@@ -29,7 +29,6 @@ coverage==4.1
 cryptography==2.0.3
 DateTime==4.1.1
 defusedxml==0.4.1
-# distribute==0.7.3 # This fails with latest pip. Installed properly as a dependency.
```

Comment:
deprecated, yes? 
https://pypi.python.org/pypi/distribute/0.6.49

Reply:
yes, like I explained in the previous comment